nixos/modules/services/networking/firefox/sync-server.nix

```diff
@@ -127,14 +127,16 @@ in
   config = mkIf cfg.enable {
 
     systemd.services.syncserver = let
-      syncServerEnv = pkgs.python.withPackages(ps: with ps; [ syncserver pasteScript requests ]);
       user = "syncserver";
       group = "syncserver";
     in {
       after = [ "network.target" ];
       description = "Firefox Sync Server";
       wantedBy = [ "multi-user.target" ];
-      path = [ pkgs.coreutils syncServerEnv ];
+      path = [
+        pkgs.coreutils
+        (pkgs.python.withPackages (ps: [ pkgs.syncserver ps.pasteScript ps.requests ]))
```

Comment:
Even though this does not look right, let's keep it that way. The bepasty module uses a similar approach.